src/Graph.ts

```diff
@@ -50,14 +63,26 @@ class SdkVersionMiddleware implements Middleware {
       throw error;
     }
   }
-
+  /**
+   * Handles setting of next middleware
+   *
+   * @param {Middleware} next
+   * @memberof SdkVersionMiddleware
+   */
   public setNext(next: Middleware): void {
     this.nextMiddleware = next;
   }
 }
-
+/**
+ * Method to reduce repetitive requests to the Graph utilized in Batch
+ *
+ * @class BatchRequest
+ */
+// tslint:disable-next-line: max-classes-per-file
```

Comment:
Let's split up the different classes and follow best practices

Reply:
478fe6f